Brain Image Library.md

```diff
@@ -1,59 +1,105 @@
 # Brain Image Library
 ## Description
-The Brain Image Library (BIL) will support the goals of the BRAIN Initiative
-Cell Census Network (BICCN) by providing a portal to the entire set of
-images submitted by the participating BICCN labs.  High-performance
-computational resources with high-bandwidth access to this library are
-available both through BIL and the XSEDE program, and will allow analysis
-of these datasets at scale.
+The Brain Image Library (BIL) is a national public resource enabling
+researchers to deposit, analyze, mine, share and interact with large brain
+image datasets.  BIL encompasses the deposition of datasets, the integration
+of datasets into a searchable web-accessible system, the redistribution of
+datasets, and a computational enclave to allow researchers to process
+datasets in-place and share restricted and pre-release datasets.  BIL is the
+designated repository for microscopy data produced through the NIH BRAIN
+Initiative.
 
 # Roles
-[Primary Investigator] Alex Ropelewski (ropelews@psc.edu)
-[System Hardware/Software] Greg Hood (ghood@psc.edu)
-[Website/Submission Portal] Luke Tuite (ltuite96@gmail.com)
-[other names to be added shortly]
+Please use bil-support@psc.edu for any Brain Image Library issues or
+questions in lieu of contacting people listed here directly.
 
-## Objectives
-* Construct and operate a large-scale storage platform suitable for housing
-  many thousands of image datasets totaling many petabytes in size.
+[Primary Investigator] Alex Ropelewski (ropelews@psc.edu)  
+[MPI – Biologic Imaging] Simon Watkins (simon.watkins@pitt.edu)  
+[MPI – Molecular Biosensors and Imaging] Marcel Bruchez (bruchez@andrew.cmu.edu)  
+[Computational Hardware/Software] Greg Hood (ghood@psc.edu)  
+[Networking Support] Kathy Benninger (benninge@psc.edu)  
+[Image Data] Arthur Wetzel (awetzel@psc.edu)  
+[Website/Submission Portal] Luke Tuite (ltuite96@gmail.com)  
+[Microscopy/Cell Biology] Alan Watson (alan.watson@pitt.edu)
 
-* Host a metadata portal to allow searching through the metadata of
-  these datasets.
 
-* Host an image portal to allow rapid access to images from these datasets.
+## Objectives
+* Permanent repository for high-quality brain microscopy datasets
+    – Brain images of mouse, rat, human, other mammals and model organisms
+    – Targeted experiments, including connectivity between cells and
```

Comment:
In some cases, like spatial transcriptomics data, repository may hold processed files instead of raw image files (although raw data is preferable).  Should we clarify that in scope is hosting of derivative files from the image data?  Morphology swc files is a similar example.